executor/executor_test.go

```diff
@@ -7811,173 +7810,6 @@ func (s *testSuite) TestIssue15563(c *C) {
 	tk.MustQuery("select distinct 0.7544678906163867 /  0.68234634;").Check(testkit.Rows("1.10569639842486251190"))
 }
 
-func (s *testSerialSuite) TestStalenessTransaction(c *C) {
```

Comment:
I moved all the staleness txn unit test to a dedicated test file from executor_test